test/internal/kubernetes/utils.go

```diff
@@ -167,15 +166,11 @@ func MustSetUpRBAC(ctx context.Context, clientset *kubernetes.Clientset, rolePat
 	roles := clientset.RbacV1().Roles(role.Namespace)
 	roleBindings := clientset.RbacV1().RoleBindings(roleBinding.Namespace)
 
-	if err = mustCreateRole(ctx, roles, role); err != nil {
+	if err := mustCreateRole(ctx, roles, role); err != nil {
 		return err
```

Comment:
nit: should we wrap these errors too ? Since we are doing it for others.

Reply:
Actually will probably leave this, most of the other methods in this file are the same